FAIL: symbol /tmp/aftershock-cpp-port/renderervk-vk/vk.c.o /tmp/aftershock-cpp-port/renderervk-vk/vk.cxx.o
--- /tmp/aftershock-cpp-port/renderervk-vk/vk.c.o
+++ /tmp/aftershock-cpp-port/renderervk-vk/vk.cxx.o
@@ -1,77 +1,3 @@
-R blend_frag_spv
-R bloom_frag_spv
-R blur_frag_spv
-R color_frag_spv
-R color_vert_spv
-R dot_frag_spv
-R dot_vert_spv
-R fog_frag_spv
-R fog_vert_spv
-R frag_light
-R frag_light_fog
-R frag_light_line
-R frag_light_line_fog
-R frag_tx0
-R frag_tx0_df
-R frag_tx0_ent
-R frag_tx0_ent_fog
-R frag_tx0_fixed
-R frag_tx0_fixed_fog
-R frag_tx0_fog
-R frag_tx0_ident1
-R frag_tx0_ident1_fog
-R frag_tx1
-R frag_tx1_cl
-R frag_tx1_cl_fog
-R frag_tx1_fixed
-R frag_tx1_fixed_fog
-R frag_tx1_fog
-R frag_tx1_ident1
-R frag_tx1_ident1_fog
-R frag_tx2
-R frag_tx2_cl
-R frag_tx2_cl_fog
-R frag_tx2_fog
-R gamma_frag_spv
-R gamma_vert_spv
-R vert_light
-R vert_light_fog
-R vert_tx0
-R vert_tx0_env
-R vert_tx0_env_fog
-R vert_tx0_fixed
-R vert_tx0_fixed_env
-R vert_tx0_fixed_env_fog
-R vert_tx0_fixed_fog
-R vert_tx0_fog
-R vert_tx0_ident1
-R vert_tx0_ident1_env
-R vert_tx0_ident1_env_fog
-R vert_tx0_ident1_fog
-R vert_tx1
-R vert_tx1_cl
-R vert_tx1_cl_env
-R vert_tx1_cl_env_fog
-R vert_tx1_cl_fog
-R vert_tx1_env
-R vert_tx1_env_fog
-R vert_tx1_fixed
-R vert_tx1_fixed_env
-R vert_tx1_fixed_env_fog
-R vert_tx1_fixed_fog
-R vert_tx1_fog
-R vert_tx1_ident1
-R vert_tx1_ident1_env
-R vert_tx1_ident1_env_fog
-R vert_tx1_ident1_fog
-R vert_tx2
-R vert_tx2_cl
-R vert_tx2_cl_env
-R vert_tx2_cl_env_fog
-R vert_tx2_cl_fog
-R vert_tx2_env
-R vert_tx2_env_fog
-R vert_tx2_fog
 T create_pipeline
 T vk_alloc_vbo
 T vk_begin_bloom_extract_render_pass
@@ -236,7 +162,81 @@
 b vk_surface
 d vkMaxSamples
 d vkSamples
+r blend_frag_spv
+r bloom_frag_spv
+r blur_frag_spv
+r color_frag_spv
+r color_vert_spv
+r dot_frag_spv
+r dot_vert_spv
+r fog_frag_spv
+r fog_vert_spv
+r frag_light
+r frag_light_fog
+r frag_light_line
+r frag_light_line_fog
+r frag_tx0
+r frag_tx0_df
+r frag_tx0_ent
+r frag_tx0_ent_fog
+r frag_tx0_fixed
+r frag_tx0_fixed_fog
+r frag_tx0_fog
+r frag_tx0_ident1
+r frag_tx0_ident1_fog
+r frag_tx1
+r frag_tx1_cl
+r frag_tx1_cl_fog
+r frag_tx1_fixed
+r frag_tx1_fixed_fog
+r frag_tx1_fog
+r frag_tx1_ident1
+r frag_tx1_ident1_fog
+r frag_tx2
+r frag_tx2_cl
+r frag_tx2_cl_fog
+r frag_tx2_fog
+r gamma_frag_spv
+r gamma_vert_spv
 r present_formats
+r vert_light
+r vert_light_fog
+r vert_tx0
+r vert_tx0_env
+r vert_tx0_env_fog
+r vert_tx0_fixed
+r vert_tx0_fixed_env
+r vert_tx0_fixed_env_fog
+r vert_tx0_fixed_fog
+r vert_tx0_fog
+r vert_tx0_ident1
+r vert_tx0_ident1_env
+r vert_tx0_ident1_env_fog
+r vert_tx0_ident1_fog
+r vert_tx1
+r vert_tx1_cl
+r vert_tx1_cl_env
+r vert_tx1_cl_env_fog
+r vert_tx1_cl_fog
+r vert_tx1_env
+r vert_tx1_env_fog
+r vert_tx1_fixed
+r vert_tx1_fixed_env
+r vert_tx1_fixed_env_fog
+r vert_tx1_fixed_fog
+r vert_tx1_fog
+r vert_tx1_ident1
+r vert_tx1_ident1_env
+r vert_tx1_ident1_env_fog
+r vert_tx1_ident1_fog
+r vert_tx2
+r vert_tx2_cl
+r vert_tx2_cl_env
+r vert_tx2_cl_env_fog
+r vert_tx2_cl_fog
+r vert_tx2_env
+r vert_tx2_env_fog
+r vert_tx2_fog
 t SHADER_MODULE
 t begin_command_buffer
 t create_color_attachment
